articles/machine_learning_engineering/experimental_trackers/Diffusion_Model_Training_Gradio.ipynb

```diff
@@ -0,0 +1,9492 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install -q -U gradio tensorboard pyarrow accelerate torchvision transformers datasets pytorch-fid diffusers
</pre><p>Let's pin each package to its current version so the code doesn't break in a few months.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/decodingml/articles-code/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>